ray-on-gke/platform/modules/gke_standard/variables.tf

```diff
@@ -51,4 +51,10 @@ variable "enable_tpu" {
   type        = bool
   description = "Set to true to create TPU node pool"
   default     = false
+}
+
+variable "tpu_node_locations" {
```

Comment:
Do we need this as a variable? Since availability is limited, maybe it would be sufficient to just hardcode it?

Reply:
yes we can also do hardcode